src/test/java/com/exasol/errorreporting/ParameterDefinitionListTest.java

```diff
@@ -0,0 +1,30 @@
+package com.exasol.errorreporting;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.sameInstance;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+class ParameterDefinitionListTest {
+    @Test
+    void testGet() {
```

Comment:
Add a test that adds two parameter definitions with the same name, assert that only the first one is returned.

Reply:
Done.